prebid-mobile/code-integration-android.md

```diff
@@ -43,9 +43,11 @@ The steps for using Prebid Mobile are as following:
 1. Create the ad units with ad unit ids and add sizes for banner ad units
 2. Add a server side configuration for each ad unit to Prebid Server Adapter
 3. Set targeting parameters for the ad units (Optional)
-4. Register the ad units with the adapter to start bid fetching process
+4. Set the primary adserver for the bid to either DFP or MoPub (the primary ad server is necessary to determine the caching mechanism)
+5. Set the host for the bid to AppNexus or Rubicon
```

Comment:
I'd prefer not to refer directly to "AppNexus" or "Rubicon" here.  Maybe we can say something like "Define the Prebid Server Host with a value from the list below" and then link to some other page which maintains a list of the available Prebid Server Host options.

Reply:
According to @ppuviarasu, right now the only serve hosts we support are AN and Rubicon. Once we have more, I think a list of available hosts makes sense, but for now it would just be an annoyance to users to click to a new page for that little information.